src/utils/checkSpelling.js

```diff
@@ -0,0 +1,110 @@
+const OriginalKeyword = require("../models/OriginalKeyword");
+
+async function getOriginalKeywords() {
+  const correctWords = await OriginalKeyword.findOne({
+    name: "originalKeywords",
+  });
+  return correctWords.value;
```

Comment:
쿼리 결과에 대한 추가적인 조작이 없어서 lean() 붙이면 좋을 것 같고, 개행이 필요해 보입니다!
```suggestion
  const correctWords = await OriginalKeyword.findOne({
    name: "originalKeywords",
  }).lean();
  
  return correctWords.value;
```

Reply:
네! 수정 반영하겠습니다!